### PR TITLE
Shortening labels for BDS terms

### PR DESCRIPTION
**Pull request by shawntanzk**
Fixes https://github.com/obophenotype/cell-ontology/issues/1536

### CODE REVIEW

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -29769,14 +29770,15 @@ AnnotationAssertion(rdfs:label obo:CL_4023116 "type 2 spiral ganglion neuron")
 SubClassOf(obo:CL_4023116 obo:CL_0011113)
 SubClassOf(obo:CL_4023116 ObjectSomeValuesFrom(obo:RO_0000053 obo:PATO_0070025))
 
-# Class: obo:CL_4023118 (L5/6 non-Martinotti morphology sst expressing GABAergic cortical internueron (Mus musculus))
+# Class: obo:CL_4023118 (L5/6 non-Martinotti sst GABAergic cortical internueron (Mmus))
```
Comment (paolaroncaglia) [inline src/ontology/cl-edit.owl:29773]: @shawntanzk internueron -> interneuron please :-)

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -29769,14 +29770,15 @@ AnnotationAssertion(rdfs:label obo:CL_4023116 "type 2 spiral ganglion neuron")
...
 
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:27477017"^^xsd:string) obo:IAO_0000115 obo:CL_4023118 "A sst GABAergic interneuron does not have Martinotti morphology with a soma found in L5/6 of the cerebral cortex.")
 AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> obo:CL_4023118 <http://orcid.org/0000-0001-7258-9596>)
-AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4023118 "L5/6 non-Martinotti SST (Mus)")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4023118 "L5/6 non-Martinotti SST (Mus musculus)")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4023118 "L5/6 non-Martinotti morphology sst expressing GABAergic cortical internueron (Mus musculus)")
```
Comment (paolaroncaglia) [inline src/ontology/cl-edit.owl:29778]: @shawntanzk internueron -> interneuron

**Review by paolaroncaglia — COMMENTED** [review]
@shawntanzk other than the typos, the diff looks ok to me. The labels were shortened only a little, but I think you can't do much more than that if you want to keep them all unambiguous. I think it's ok to merge once the typos are fixed. Cheers.

File: src/ontology/cl-edit.owl (hunk 2 above)
Comment (shawntanzk) [reply src/ontology/cl-edit.owl:29778]: whoops >.< thanks for the catch

**Pull request merged**